app/controllers/filter_data_controller.rb

```diff
@@ -9,7 +9,7 @@ class FilterDataController < ApplicationController
   def qings
     authorize!(:index, Response)
     qings = Questioning.for_mission(current_mission).joins(:question)
-      .includes(:question).order("questions.code")
+      .includes(:question).where(question: Question.reportable).order("questions.code")
```

Comment:
Oh, this is not what I meant, but good find! 

Please see app/models/question_type.rb. 

We should refactor this old scope in fact. Can you add another property (`reportable`) to `AVAILABLE_PROPERTIES` and then add a new scope to Question called `with_property(property)`? Then wherever `reportable` is used, use that instead. 

And then for your purposes here, please use the `refable` property because that also excludes location questions, which is appropriate here. `refable` is also used with the condition system, which you're reusing in the filters.

Reply:
I think I understand, and I took a whack at it, but the method delegation doesn't seem to work. Let's pair tomorrow or Fri.

```
undefined method `with_property' for #<Questioning::ActiveRecord_Relation>
```